src/sentry/eventstream/snuba.py

```diff
@@ -111,7 +111,7 @@ def insert(
                     "project_id": event.project_id,
                     # TODO(mitsuhiko): We do not want to send this incorrect
                     # message but this is what snuba needs at the moment.
-                    "message": event.message,
+                    "message": event.search_message,
```

Comment:
Wait, either I am missing something or the current `search_message` does not really look like what `message` looked like before PR 16557. 

This was message before the PR
```
 def message(self):
    if hasattr(self, "_message"):
        return self._message
    column = self.__get_column_name(Columns.MESSAGE)
    if column in self._snuba_data:
        return self._snuba_data[column]
    return self.real_message
```

How was the content of search_message being stored into message ?

Reply:
It was saved into the message field with this line in that previous PR.
```event.message = event.search_message```
It previously needed to be called event.message so it would be compatible with the django model.

Reply:
ok. did not see that.